exercises/rotational-cipher/src/example/kotlin/RotationalCipher.kt

```diff
@@ -0,0 +1,18 @@
+class RotationalCipher(val shiftKey: Int) {
+
+    companion object {
+        private val LOWERS = "abcdefghijklmnopqrstuvwxyz"
+        private val UPPERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Or how about:
```
private val LOWERS = ('a' .. 'z').toList()
private val UPPERS = ('A' .. 'Z').toList()
```

I am cool with any of these options the rest of this pull request looks good, merge away whichever way you want to go.